scripts/package.ps1

```diff
@@ -38,6 +40,16 @@ foreach ($file in [System.IO.Directory]::EnumerateFiles("$repoRoot\src", "*.cspr
     }
 }
 
+if ($Sign)
+{
+    Write-Host "** Signing packages with SignType = $SignType **"
+    & $dotnetExePath msbuild "$repoRoot\tools\sign.proj" /p:SignType="$SignType" /p:PackagesPath="$packagesPath"
+
+    if (!$?) {
+        exit $lastExitCode
```

Comment:
Nit: Write an error message before exiting

`Write-Error "Failed to sign the packages"`